### PR TITLE
Add "Anchor permanence" policy

### PR DESCRIPTION
**Pull request by domenic**
Part of whatwg/meta#84.

@whatwg/editors thoughts? Does this seem reasonable?

<del>(This is on top of #71. Please only review the last commit. I will rebase when this is merged.)</del><ins>(This has been rebased and is now a single commit.)</ins>

### CODE REVIEW

File: working-mode [hunk 1]
```diff
@@ -287,6 +287,56 @@ pragmatic solution in case of conflict.
...
+can mislead implementers and web developers.
+
+<p>For cases where another standards organization wishes to ensure an anchor is permanently
+available in the canonical Living Standard, they may file a new issue requesting a permanent anchor
+and detailing the anchor(s) they would like preserved. The editor can then discuss this request with
+the requestor; for example, if the editor was planning on making changes in the near future, maybe
+they would advise the requestor to hold off on linking to that anchor until the changes go through.
+
+<p>After discussion concludes, such requested anchors get promoted to be permanent. This means that
+it becomes a matter of WHATWG Policy to never break them without the requestor's assent. (This is
```
Comment (domenic) [inline working-mode]: Absolutely requiring the requestor's assent is perhaps a bit too strong. Suggestions welcome.

File: working-mode [hunk 2]
```diff
@@ -287,6 +287,56 @@ pragmatic solution in case of conflict.
...
+<a href="https://console.spec.whatwg.org/#console-namespace">#console-namespace</a>, the old anchor
+of <a href="https://console.spec.whatwg.org/#console-interface">#console-interface</a> still
+works.</p>
+
+<p class="example">Although the concept of "structured cloning" has been replaced by structured
+serialization, the HTML Standard's
+<a href="https://html.spec.whatwg.org/multipage/structured-data.html#structured-clone">#structured-clone</a>
+anchor points to a note explaining this evolution, so that old links do not break.</p>
+
+<p>Consumers who need to reference exactly what a Living Standard said at a given point in time,
```
Comment (domenic) [inline working-mode]: Not sure if this paragraph belongs here; thoughts welcome.

**Comment by othermaciej** [comment]
This seems to contain a lot of reformatting that isn't related to the change. Also totally reformats code-of-conduct, which I think is not even included in the change. Is there a way to separate out the non-substantive formatting changes into a separate PR for ease of review?

**Comment by domenic** [comment]
Yes, in fact I did that :). See #71. Only the last commit needs to be reviewed here. (Click it to see the diff.)

**Comment by othermaciej** [comment]
I see, that is indeed much better.  I foolishly looked at the Files view instead.

File: working-mode (hunk 2 above)
Comment (annevk) [reply working-mode]: I think that's more what we want from others and not really how we do work. I'm not sure what a better place would be though so we might want to keep it here for now.

File: working-mode (hunk 1 above)
Comment (annevk) [reply working-mode]: I think it's fine, but ideally the requestor is then some formal body so there'll be a point of contact. I don't think we want to allow this for blog posts and such, for instance (though we'll try).

**Comment by annevk** [comment]
I wonder if we should mention the Bikeshed/Shepherd protocol since if you use that, you don't really have to worry about this.

File: working-mode [hunk 3]
```diff
@@ -287,6 +287,56 @@ pragmatic solution in case of conflict.
...
 features.
 
+<h3 id="anchors">Anchor permanence<a class="self-link" href="#anchors"></a></h3>
+
+<p>Often other standards want to reference parts of a WHATWG Living Standard by using a hypertext
+anchor. Since Living Standards are continually evolving, the set of anchors in a document is not
+static, and some anchors could disappear over time.
+
+<p>In general, editors should try to preserve anchors when reasonable. This includes using
+techniques like:
```
Comment (domfarolino) [inline working-mode]: Definitely +1 to this

File: working-mode [hunk 4]
```diff
@@ -287,6 +287,56 @@ pragmatic solution in case of conflict.
...
+serialization, the HTML Standard's
+<a href="https://html.spec.whatwg.org/multipage/structured-data.html#structured-clone">#structured-clone</a>
+anchor points to a note explaining this evolution, so that old links do not break.</p>
+
+<p>Consumers who need to reference exactly what a Living Standard said at a given point in time,
+for example to document what they implemented from or to explain the history of the web platform,
+can use the commit snapshots functionality. Each Living Standard has a "Snapshot as of this commit"
+link that gives a frozen copy for such historical reference. The WHATWG will keep these snapshots
+available at their published URLs permanently. However, other standards organizations are
+discouraged from referencing these snapshots, as they generally contain contain known issues that
```
[the remark this answers is not in the document]
Comment (domenic) [reply working-mode]: Done

**Comment by domenic** [comment]
Rebased on master. Any further SG or editor thoughts? Shall we merge?

**Comment by othermaciej** [comment]
Is there a way to tag the whole set of editors to see if there are other opinions on this change?

**Comment by domfarolino** [comment]
@othermaciej Indeed, I believe he did that with @whatwg/editors a bit earlier on, not sure who all has seen it though.

**Comment by othermaciej** [comment]
Since editors who have spoken up are comfortable with this change, I am ok with this. Since this is a substantive policy change, it should have sign-off from other SG representatives. Tagging @henceproved @michaelchampion @dbaron.

**Comment by henceproved** [comment]
The change looks good to me too.

**Comment by michaelchampion** [comment]
Looks good to me as well

**Comment by annevk** [comment]
@domenic you didn't really reply to my points about the Shepherd/Bikeshed protocol and being able to dismiss certain requests.

(I also pushed a small nit commit.)

**Comment by domenic** [comment]
Looks like @annevk is OK with my response, so this is just waiting @dbaron's approval and for a SG member to approve/merge :)

**Comment by othermaciej** [comment]
3/4 SG approvals is enough. Go ahead and merge.

**Pull request merged**